ui/src/data/countries.json

```diff
@@ -0,0 +1,246 @@
+[
+  { "name": "Afghanistan", "code": "AF" },
```

Comment:
This is fine for now. Do you think we'll ever need to put this in the backend?

Reply:
I don't think a countries/cities list is ever used in the backend since we use Mapbox to query the coordinates